Can't build on Fedora 24

When running `cmake` I've got this error:
```
CMake Error at CMakeLists.txt:19 (find_package):
  By not providing "FindXdgUserDirs.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "XdgUserDirs", but CMake did not find one.

  Could not find a package configuration file provided by "XdgUserDirs" with
  any of the following names:

    XdgUserDirsConfig.cmake
    xdguserdirs-config.cmake

  Add the installation prefix of "XdgUserDirs" to CMAKE_PREFIX_PATH or set
  "XdgUserDirs_DIR" to a directory containing one of the above files.  If
  "XdgUserDirs" provides a separate development package or SDK, be sure it
  has been installed.


-- Configuring incomplete, errors occurred!
```

Got any tips where I can find this cmake script?
I've tried to search it with command `dnf whatprovides '*/XdgUserDirsConfig.cmake'` with no luck.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,6 @@ find_package(Qt5Widgets REQUIRED QUIET)
 find_package(Qt5LinguistTools REQUIRED QUIET)
 
 find_package(lxqt REQUIRED QUIET)
-find_package(XdgUserDirs REQUIRED QUIET)
 
 include(LXQtCompilerSettings NO_POLICY_SCOPE)
 include(LXQtTranslate)

--- a/README.md
+++ b/README.md
@@ -13,7 +13,7 @@ The software belongs to the LXQt project but its usage isn't limited to this des
 ### Compiling source code
 
 Runtime dependencies are qtbase and PulseAudio client library libpulse.   
-Additional build dependencies are CMake, xdg-user-dirs and [liblxqt](https://github.com/lxde/liblxqt) as well as optionally Git to pull latest VCS checkouts. The localization files were outsourced to repository [lxqt-l10n](https://github.com/lxde/lxqt-l10n) so the corresponding dependencies are needed, too. Please refer to this repository's `README.md` for further information.   
+Additional build dependencies are CMake and [liblxqt](https://github.com/lxde/liblxqt) as well as optionally Git to pull latest VCS checkouts. The localization files were outsourced to repository [lxqt-l10n](https://github.com/lxde/lxqt-l10n) so the corresponding dependencies are needed, too. Please refer to this repository's `README.md` for further information.
 
 Code configuration is handled by CMake. CMake variable `CMAKE_INSTALL_PREFIX` has to be set to `/usr` on most operating systems.   
 

